.travis.yml

```diff
@@ -0,0 +1,20 @@
+language: python
+
+python:
+  - 2.7
+  - 3.4
```

Comment:
I would test CPython 3.7 too.  If you want to keep it to five jobs for perf reasons (I can see an argument each way), I think 3.4 is the version to omit as it goes EOL in March 2019.

Reply:
I m not really concerned with perfs to be honest, only with accurate algorithmic complexity ;-)
FWIW the test on 3.4 reminded me that async syntax was not yet supported on that version. So I think it is worth keeping around.